src/main/java/it/project1/item/ItemRepositoryOld.java

```diff
@@ -0,0 +1,172 @@
+/* package it.project1.item;
```

Comment:
se lo hai commentato e non ti serve più togliamolo che credo sia meglio, altrimenti potrebbe darti problemi in futuro